RxAtomic/include/RxAtomic.h

```diff
@@ -0,0 +1,48 @@
+//
+//  RxAtomic.h
+//  RxAtomic
+//
+//  Created by Krunoslav Zaher on 10/28/18.
+//  Copyright © 2018 Krunoslav Zaher. All rights reserved.
+//
+
+#ifndef RxAtomic_h
+#define RxAtomic_h
+
+#include <stdatomic.h>
+
+#define SWIFT_NAME(_name) __attribute__((swift_name(#_name)))
+
+#define Atomic(swift_type, llvm_type) \
+    typedef struct { volatile atomic_##llvm_type atom; } Atomic##swift_type;\
+    static __inline__ __attribute__((__always_inline__)) SWIFT_NAME(Atomic##swift_type.initialize(self:_:)) \
+    void Atomic##swift_type##_Initialize(Atomic##swift_type * _Nonnull self, llvm_type value) { \
+        atomic_init(&self->atom, value);\
+    }\
+    \
+    static __inline__ __attribute__((__always_inline__)) SWIFT_NAME(Atomic##swift_type.load(self:)) \
+    llvm_type Atomic##swift_type##_Load(Atomic##swift_type * _Nonnull self) { \
```

Comment:
Oh boy... this is a bit of a crazy hack in a way - I guess there's no other way using stdatomic from Swift code that you know of ?

Reply:
The other was was `OSAtomic*` :)